app/models/user.rb

```diff
@@ -101,6 +101,11 @@ def pending_profile
     profiles.gpo_verification_pending.order(created_at: :desc).first
   end
 
+  def personal_key_generated_at
+    self.encrypted_recovery_code_digest_generated_at ||
+      active_profile&.verified_at
```

Comment:
We could leave out `self.` here in referencing the instance method.

```suggestion
    encrypted_recovery_code_digest_generated_at || active_profile&.verified_at
```